src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java

```diff
@@ -86,18 +90,22 @@ public final Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest b
     }
 
     private List<ExtImpNextMillennium> getImpExts(BidRequest bidRequest, List<BidderError> errors) {
-        final List<ExtImpNextMillennium> impExts = new ArrayList<>();
-
-        for (Imp imp : bidRequest.getImp()) {
-            try {
-                impExts.add(mapper.mapper()
-                        .convertValue(imp.getExt(), NEXTMILLENNIUM_EXT_TYPE_REFERENCE)
-                        .getBidder());
-            } catch (IllegalArgumentException e) {
-                errors.add(BidderError.badInput(e.getMessage()));
-            }
+        return bidRequest.getImp()
+                .stream()
```

Comment:
Nitpick. We prefer placing call to `.stream()` on first line: 
```
return bidRequest.getImp().stream()
...
.toList()
```

Reply:
@And1sS fixed